week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,31 +1,47 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// The Math. random() function is used to return a floating-point pseudo-random number between range [0,1)
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
+console.log(getNumber());
 
 // Add comments to explain what this function does. You're meant to use Google!
+// The concatenate function is one of Excel's text functions.
```

Comment:
Concat is not one of the Exel's text functions, it is one of the String methods in javascript. Wrong explanation here.